snitchpy/__init__.py

```diff
@@ -285,6 +293,23 @@ async def call():
         self.audiences[self._aud_to_str(aud)] = aud
         self.grpc_loop.run_until_complete(call())
 
+    def _add_audiences(self) -> None:
+        """This method is used to re-announce audiences after a disconnect"""
+
+        async def call():
+            try:
+                req = protos.NewAudienceRequest(
+                    audience=aud, session_id=self.session_id
+                )
+                await self.grpc_stub.new_audience(
+                    req, timeout=self.grpc_timeout, metadata=self._get_metadata()
+                )
+            except Exception:
+                pass
```

Comment:
I don't remember what the logger in python was like - would be great if one could be injected (same as `snitch-go-client`) and then we can instrument all these exception catches to log a debug or warnf msg.

Reply:
Added debug line